src/Framework/Support/ValueObjects/BaseEnum.php

```diff
@@ -0,0 +1,324 @@
+<?php
+
+namespace Give\Framework\Support\ValueObjects;
+
+/**
+ * @unreleased remove stringable
+ *
+ * This is a fork of the myclabs/php-enum library 1.8.4.
+ * In GiveWP we have php constraints that do not allow us to use the latest version of this package.
```

Comment:
Let's detail what the constraints are so we know when we're no longer bound by them.

Reply:
Sounds good cea4bf4b